maps/site53/job/jobs/misc.dm

```diff
@@ -41,9 +44,8 @@
 	//supervisors = "the Archivist and administrative staff"
 	//duties = "<big><b>You are a low level pen pusher of the SCP Foundation. Your direct supervisor is the Archivist but you may also answer to the O5 and Ethics Committee Representative. Write reports, assist researchers and generally be a pain in the ass to everyone around you.</big></b>"
 	economic_power = 2
-	minimal_player_age = 5
-	ideal_character_age = 30
-	alt_titles = list("Administrative Assistant", "Accountant", "Auditor", "Secretary")
+	ideal_character_age = 21
+	alt_titles = list("Administrative Assistant", "Accountant", "Auditor", "Secretary", "Assistant")
```

Comment:
```suggestion
	alt_titles = list("Administrative Assistant", "Accountant", "Auditor", "Secretary")
```
"Assistant" isn't a specific term, and it's outside the scope of the Office Worker (Assistant implies general assistant, which is covered by other roles e.g. junior engineer/scientist).